packages/io/svg-deserializer/toArray.js

```diff
@@ -0,0 +1,9 @@
+/* converts input data to array if it is not already an array */
```

Comment:
we do not have this somewhere already ?

Reply:
Several actually... which is why I wasn't sure what would be appropriate.
- packages/core/utils/arrays.js
- packages/io/io-utils/arrays.js

I would prefere a common set of 'utils' that house our special routines for JavaScript compatibility / interoperability. We have several sets now, but maybe it's time to decide where these live going forward.

Maybe packages/utils should house these functions?

Reply:
I agree that it should be centralized but 'utils' is too vague, should be specific (to avoid having utils be a vague bag of everything): ie **arrayUtils** makes sense: I will add a note, we can tackle this later :)